webcomponents/core/src/components/reveal/deckdeckgo-reveal/deckdeckgo-reveal.tsx

```diff
@@ -21,35 +36,18 @@ export class DeckdeckgoReveal {
       this.visibleIndex++;
 
       const elements: Node[] = Array.from(this.el.childNodes).filter((node: Node) => {
-        return node && node.nodeType !== 3;
+        return node && node.nodeType !== node.TEXT_NODE;
```

Comment:
Was there a compatibility reason for this? Enum is more clear. 😉

Reply:
Totally